### ISSUE TITLE
Order files by last modified time

### ISSUE THREAD

**Issue by dmnd** [atom/tree-view#405]
TL;DR: Feature request: option for certain directories for ordering files in the tree view by last modified time

Rationale: Some "projects" I have are just a collection of tenuously related files. E.g. a directory of adhoc database queries, or a directory of textual notes. For these directories I'd prefer if the most recently edited files were at the top of the tree view, because it's easier for me to remember I want something recent instead of the specific name I gave the file. 

There's scope to get fancy with the UI and have subsections for yesterday, last week, last month, etc.

A workaround for this is to use a Finder window but it'd be nice to have it be part of the tree view. Maybe this is a package idea though.

**Comment by mathiasvr** [comment]
Related to issue #343

**Comment by kenyarmosh** [comment]
+ 1

**Comment by saaa-hsjgrobler** [comment]
+1

**Comment by eliten00b** [comment]
+1

**Comment by pawelkuras** [comment]
+1

**Comment by lee-dohm** [comment]
This appears to be more than just related to #343. It appears to be a duplicate. Or am I missing something?

**Comment by lee-dohm** [comment]
Closing this as duplicate. Please subscribe to #343 for updates :grinning:
